ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -234,19 +235,22 @@ SQUtils.QObject {
 
         stackView: root.stackView
         keycardState: root.keycardState
+        pinSettingState: root.pinSettingState
+        authorizationState: root.authorizationState
         addKeyPairState: root.addKeyPairState
-        seedWords: root.seedWords
+        getSeedWords: root.getSeedWords
         displayKeycardPromoBanner: root.displayKeycardPromoBanner
         isSeedPhraseValid: root.isSeedPhraseValid
 
         keycardPinInfoPageDelay: root.keycardPinInfoPageDelay
 
         onReloadKeycardRequested: root.reloadKeycardRequested()
         onKeycardFactoryResetRequested: root.keycardFactoryResetRequested()
-        onKeyPairTransferRequested: root.keyPairTransferRequested()
+        onLoadMnemonicRequested: root.loadMnemonicRequested()
         onKeycardPinCreated: (pin) => root.keycardPinCreated(pin)
         onLoginWithKeycardRequested: loginWithKeycardFlow.init()
-        onKeypairAddTryAgainRequested: root.keyPairTransferRequested() // FIXME?
+        // onKeypairAddTryAgainRequested: root.keyPairTransferRequested() // FIXME?
```

Comment:
Just remove it if it's indeed not needed

Reply:
Fixed with a call to `loadMnemonicRequested`, this should foloow the designs.